basics/units.ipynb

```diff
@@ -0,0 +1,403 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<center><h1><b><span style=\"color:blue\">Playing with the HEP system of units</span></b></h1></center>\n",
+    "\n",
+    "**Or, an intro to the `hepunits` package while performing some time-of-flight studies**.\n",
+    "\n",
+    "Scikit-HEP packages used:\n",
+    "- [`hepunits`](https://github.com/scikit-hep/hepunits)\n",
+    "- [`Particle`](https://github.com/scikit-hep/particle)\n",
+    "\n",
+    "Author:\n",
+    "[Eduardo Rodrigues](https://github.com/eduardo-rodrigues/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<center><h2><b><span style=\"color:green\">hepunits - The HEP system of units</span></b></h2></center>\n",
+    "\n",
+    "The package ``hepunits`` collects the most commonly used units and constants in the\n",
+    "HEP System of Units, which are *not* the same as the international system of units (aka SI units).\n",
```

Comment:
I think what you mean is that the HEP system is basically SI, but we use assign a scale of 1 to different quantities, to mm instead of meter, to MeV instead of Joule, etc.

Reply:
I updated the wording. It is partially an implementation detail but it still is important to note that there are differences (we care more about the energy than the mass, for example).